client/my-sites/checkout/checkout-thank-you/jetpack-checkout-siteless-thank-you-completed.tsx

```diff
@@ -69,39 +94,86 @@ const JetpackCheckoutSitelessThankYouCompleted: FC< Props > = ( {
 								: 'jetpack-checkout-siteless-thank-you-completed__main-message'
 						}
 					>
-						{ translate( 'Your %(productName)s subscription will be activated soon', {
-							args: {
-								productName,
-							},
-						} ) }
+						{ title }
 					</h1>
-					<p>
-						{ translate(
-							'As soon as your subscription is activated you will receive a confirmation email from our Happiness Engineers.'
-						) }
-					</p>
-					<p>
-						{ translate(
-							'If you prefer to setup Jetpack with the help of our Happiness Engineers, {{a}}schedule a 15 minute call now{{/a}}.',
-							{
-								components: {
-									a: (
-										<a
-											className="jetpack-checkout-siteless-thank-you-completed__link"
-											onClick={ () =>
-												dispatch(
-													recordTracksEvent( 'calypso_siteless_checkout_happiness_link_clicked', {
-														product_slug: productSlug,
-													} )
-												)
-											}
-											href={ happinessAppointmentLink }
-										/>
-									),
-								},
-							}
-						) }
-					</p>
+					{ ! automaticTransferSucceeded && (
+						<>
+							<p>
+								{ translate(
+									'As soon as your subscription is activated you will receive a confirmation email from our Happiness Engineers.'
+								) }
+							</p>
+							<p>
+								{ translate(
+									'If you prefer to setup Jetpack with the help of our Happiness Engineers, {{a}}schedule a 15 minute call now{{/a}}.',
+									{
+										components: {
+											a: (
+												<a
+													className="jetpack-checkout-siteless-thank-you-completed__link"
+													onClick={ () =>
+														dispatch(
+															recordTracksEvent(
+																'calypso_siteless_checkout_happiness_link_clicked',
+																{
+																	product_slug: productSlug,
+																}
+															)
+														)
+													}
+													href={ happinessAppointmentLink }
+												/>
+											),
+										},
+									}
+								) }
+							</p>
+						</>
+					) }
+					{ automaticTransferSucceeded && (
+						<>
+							<p>
+								{ productName &&
+									rawSite?.name &&
+									translate(
+										"We successfully activated %(productName)s on {{strong}}%(siteName)s{{/strong}}. Next, we'll recommend features based on your goals",
```

Comment:
The last sentence is missing a period at the end. Same for the other instances below.

Reply:
Good catch. Thanks!